website_snippet_product_category/static/src/js/frontend.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+odoo.define("website_snippet_product_category.s_product_category", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var sAnimation = require("website.content.snippets.animation");
+
+    var _t = core._t;
+
+    sAnimation.registry.js_product_category = sAnimation.Class.extend({
```

Comment:
Is this for lazy/asynchronous rendering? If so, please indicate it as comment in the corresponding method. Remember, always docstring of the why, not the what (of the code).